src/app/design/frontend/base/default/template/payment/form/omise/omiseoffsiteconveniencestore.phtml

```diff
@@ -36,8 +36,13 @@
                 name="payment[email]"
                 title="<?php echo $this->__('Email'); ?>"
                 value="<?php echo $this->escapeHtml($this->getOmiseConvenienceStoreEmail()); ?>"
-                maxlength="10"
-                class="input-text required-entry validate-email" />
+                maxlength="50"
+                class="input-text required-entry validate-email validate-email-plussign validate-length maximum-length-50 minimum-length-6" />
```

Comment:
It feels like adding just single validation function for 'convenience store' emails, and remove `validate-email` here would be a better solution - we could then put all the rules in one function for what makes a 'valid' email in this case.

Either that, or rename `validate-email-plussign` as something directly related to convenience store emails